src/framework/ui/uitypes.h

```diff
@@ -187,47 +189,89 @@ enum class Checkable {
     Yes
 };
 
+enum class ActionCategory {
+    Undefined = -1,
+    Internal,
+    Tablature,
+    Viewingnavigation,
+    Playback,
+    Layoutformatting,
+    Selectingediting,
+    Application,
+    Accessibility,
+    File,
+    Selectionnavigation,
+    Textlyrics,
+    Chordsymbolsfiguredbass,
+    Measures,
+    Musicalsymbols,
+    Dialogspanels,
+    Noteinput,
+    Workspace,
+    Plugins
+};
+
 struct UiAction
 {
+    inline static QStringList categories
+        = { "Undefined", "Internal", "Tablature", "Viewing & Navigation", "Playback", "Layout & Formatting", "Selecting & Editing",
```

Comment:
Lists values should be written out on separate lines like the enum values above to make for easier reading, scrolling and commenting. They also need to be translated:
```c++
{
    TranslatableString("framework/actioncategory", "Undefined"),
    TranslatableString("framework/actioncategory", "Internal"), // Now I can comment on this value
    TranslatableString("framework/actioncategory", "Tablature"),
    // TranslatableString("framework/actioncategory", "Viewing & Navigation"), // And comment out this value altogether
    ...
}
```
But I'm not sure if these strings should really be in a static list or just return values of a switch statement? (Maybe another dev can comment on this?) Also, they should probably be defined in a `.cpp` file along with the body of the `getCategory()` function.